LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2023] [Red Hat]
```

Comment:
I looked at some examples in other code repositories.  It appears that we don't need to include anything after the line that states END OF TERMS AND CONDITIONS.  If we want to keep the boiler plate example then we probably should remove the [ ] around the text as indicated in the instructions.

Reply:
Thanks for noticing that.  I see that AWX just removes this whole section, so I'll do the same.